b2fuse/filetypes/B2HashFile.py

```diff
@@ -53,4 +50,4 @@ def write(self, offset, data):
         return
 
     def read(self, offset, length):
-        return self.data.tostring()
+        return bytes(self.data)
```

Comment:
I am still trying to understand what exactly you mean here.

Reply:
So I had this issue where a file that contained only ascii text would not read properly unless I used toString(). Can you verify that when writing an ascii file and reading it back you get the same out as in?

Reply:
Hi,

I tried to test this right now by running b2fuse with the `-enable_hasfiles` option.
It seems as though the `B2HashFile` already crashes in the `__ini__` method, when trying to access the `contentSha1` key in the `file_info` dict.
This error should also occur on the current master branch and maybe was introduced by an API update of backblaze?

Reply:
@sondree Can you confirm if the backblaze SDK is no longer exposeing `contentSha1` via `file_info`?
If so, do you see another way of obtaining the `contantSha1` information, or would you suggest just removing that feature?

Reply:
@sondree Not sure if you saw this comment or not? I know you're busy and I don't expect you to answer, just wanting to make sure you're not missing it.

Reply:
It seems to me that it is still exposed through b2_list_file_names (https://www.backblaze.com/b2/docs/b2_list_file_names.html). Is this not the case? I do not know why it would crash on the top of my head.

Reply:
It looks like the problem was caused by a change in the b2 python library.
It took some digging but I found a workaround: https://github.com/sondree/b2_fuse/pull/67/commits/2858c34fe0349a5a87ad87c10fbeccac4e613569#diff-ed85c3b14e7e379241f4d59f769b370fR131

Reply:
I also tested your original concern regarding the ASCII files. I was able to read a hash file with only ascii text. I think this works fine.

Reply:
Sweet, thanks!